dokka-runners/dokka-gradle-plugin/settings.gradle.kts

```diff
@@ -2,7 +2,7 @@
  * Copyright 2014-2024 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
 
-rootProject.name = "runner-gradle-plugin-classic"
+rootProject.name = "dokka-gradle-plugin"
```

Comment:
I renamed the directory because Gradle recommends keeping the artifact ID and the project name aligned. It also makes the publishing configuration simpler.